client/src/components/Profile/Profile.js

```diff
@@ -1,17 +1,64 @@
+import { useEffect, useState } from 'react'
+import axios from 'axios'
 import styled from 'styled-components/macro'
 import LinkButtonBlue from '../Buttons/LinkButtonBlue/LinkButtonBlue'
-import LinkButtonGreen from '../Buttons/LinkButtonGreen/LinkButtonGreen'
+import ButtonGreen from '../Buttons/ButtonGreen/ButtonGreen'
 import LinkButtonRed from '../Buttons/LinkButtonRed/LinkButtonRed'
+import LoadingSpinner from '../Messages/LoadingSpinner/LoadingSpinner'
+import ErrorCard from '../Messages/ErrorCard/ErrorCard'
+
+const Profile = ({ onLogout }) => {
+  const [userData, setUserData] = useState('')
+  const [isLoading, setIsLoading] = useState(true)
+  const [errorMessage, setErrorMessage] = useState('')
+
+  useEffect(() => {
+    const fetchUser = async () => {
+      try {
+        const token = JSON.parse(localStorage.getItem('authToken'))
```

Comment:
As far as I did understand Lenes comment it we should extract the localStorage functions. For me this means that you should import the get function and then use the imported function here. 🤔